bot/constants.py

```diff
@@ -774,3 +774,9 @@ class ThreadArchiveTimes(Enum):
     "Noooooo!!",
     "I can't believe you've done this",
 ]
+
+DEFAULT_RULES_DESCRIPTION = (
+    "The rules and guidelines that apply to this community can be found on"
+    " our [rules page](https://www.pythondiscord.com/pages/rules). We expect"
+    " all members of the community to have read and understood these."
+)
```

Comment:
Sorry for the back-and-forth and not being clearer about this. The constant should be defined in `information.py`. The `bot/constants.py` file is meant for global constants i.e. constants shared by many modules. Tests don't really fit the criteria of "shared by many modules".